src/stream/src/executor/aggregate/agg_group.rs

```diff
@@ -383,12 +384,24 @@ impl<S: StateStore, Strtg: Strategy> AggGroup<S, Strtg> {
         if self.curr_row_count() == 0 {
             tracing::trace!(group = ?self.ctx.group_key_row(), "first time see this group");
         }
-        for (((state, call), func), visibility) in (self.states.iter_mut())
-            .zip_eq_fast(calls)
-            .zip_eq_fast(funcs)
-            .zip_eq_fast(visibilities)
-        {
-            state.apply_chunk(chunk, call, func, visibility).await?;
+
+        let concurrency = 10;
```

Comment:
or try_join_all at once,

I saw a few other places that just try all of them in agg